charts/infra-apps/Chart.yaml

```
@@ -15,35 +15,23 @@ dependencies:
     version: 0.9.1
     repository: https://charts.adfinis.com
 annotations:
-  artifacthub.io/containsSecurityUpdates: "true"
+  artifacthub.io/containsSecurityUpdates: "false"
   artifacthub.io/changes: |
     - kind: changed
       description: |
-        feat!: Update kube-prom-stack from 72.5.0 to 75.3.0
+        feat: Update cert-manager from 1.17.1 to 1.18.1
```

```suggestion
        feat!: Update cert-manager from 1.17.1 to 1.18.1
```
It says breaking change later on, right?